components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineView.kt

```diff
@@ -126,10 +127,9 @@ class SystemEngineView @JvmOverloads constructor(
             url?.let {
                 val cert = view?.certificate
                 session?.internalNotifyObservers {
-                    onLocationChange(it)
```

Comment:
I think this should still be called here. We shouldn't introduce multiple ways of how the URL gets updated.

Reply:
👍